### PR TITLE
TSK-1130

### PR DESCRIPTION
**Pull request by gitgoodjhe**
-Updated deprecated method names and
-Updated deprecated javadoc
-Added BEANS to configurations
-removed the @author tag everywhere, see this: https://www.vojtechruzicka.com/stop-using-javadoc-author-tag/
-removed unnecessary files/scripts

### CODE REVIEW

File: taskana-adapter-camunda-listener/src/main/java/pro/taskana/adapter/camunda/parselistener/TaskanaParseListenerProcessEnginePlugin.java [hunk 1]
```diff
@@ -19,9 +19,8 @@
 import pro.taskana.adapter.camunda.util.ReadPropertiesHelper;
 
 /**
- * Camunda engine plugin for the taskana parse listener.
- *
- * @author jhe
+ * Camunda engine plugin for the responsible for adding the TaskanaParseListener to the
```
Comment (BerndBreier) [inline taskana-adapter-camunda-listener/src/main/java/pro/taskana/adapter/camunda/parselistener/TaskanaParseListenerProcessEnginePlugin.java]: should read "Camunda engine plugin responsible for adding the TaskanaParseListener ..."

File: taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/TaskanaConfigurationProperties.java [hunk 2]
```diff
@@ -2,19 +2,17 @@
...
+  String TASKANA_ADAPTER_OUTBOX_DATASOURCE_DRIVER =
       "taskana.adapter.outbox.datasource.driver";
-  static final String TASKANA_ADAPTER_OUTBOX_DATASOURCE_URL =
+  String TASKANA_ADAPTER_OUTBOX_DATASOURCE_URL =
       "taskana.adapter.outbox.datasource.url";
-  static final String TASKANA_ADAPTER_OUTBOX_DATASOURCE_USERNAME =
+  String TASKANA_ADAPTER_OUTBOX_DATASOURCE_USERNAME =
       "taskana.adapter.outbox.datasource.username";
-  static final String TASKANA_ADAPTER_OUTBOX_DATASOURCE_PASSWORD =
+  String TASKANA_ADAPTER_OUTBOX_DATASOURCE_PASSWORD =
```
Comment (BerndBreier) [inline taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/TaskanaConfigurationProperties.java]: why is this file formatted differently than the TaskanaConfigurationProperites from the listener project?
I think, these files should be identical.

File: taskana-adapter-camunda-spring-boot-test/src/test/java/pro/taskana/adapter/integration/TestTaskAcquisition.java [hunk 3]
```diff
@@ -268,7 +267,8 @@ private void retrieveTaskanaTaskAndVerifyTaskVariables(
       Assert.assertThat(assumedVariablesString, SameJSONAs.sameJSONAs(taskanaVariablesString));
 
     } catch (TaskNotFoundException | NotAuthorizedException e) {
-      LOGGER.info("Caught {}, while trying to create a taskana task and verify its variables", e, e);
+      LOGGER.info(
+          "Caught {}, while trying to create a taskana task and verify its variables", e, e);
```
Comment (BerndBreier) [inline taskana-adapter-camunda-spring-boot-test/src/test/java/pro/taskana/adapter/integration/TestTaskAcquisition.java]: If the message contains curly braces, there should be one replacement parameter: e.getClass().getName()

File: taskana-adapter-camunda-system-connector/src/main/java/pro/taskana/adapter/systemconnector/camunda/api/impl/CamundaTaskRetriever.java [hunk 4]
```diff
@@ -50,12 +46,12 @@
...
   }
 
-  public List<ReferencedTask> retrieveTerminatedCamundaTasks(String camundaSystemUrl) {
+  public List<ReferencedTask> retrieveFinishedCamundaTasks(String camundaSystemUrl) {
     LOGGER.debug("entry to retrieveFinishedCamundaTasks. CamundSystemURL = {} ", camundaSystemUrl);
 
     List<CamundaTaskEvent> camundaTaskEvents =
         getCamundaTaskEvents(
-            camundaSystemUrl, CamundaSystemConnectorImpl.URL_GET_CAMUNDA_COMPLETE_EVENTS);
+            camundaSystemUrl, CamundaSystemConnectorImpl.URL_GET_CAMUNDA_FINISHED_EVENTS);
```
Comment (BerndBreier) [inline taskana-adapter-camunda-system-connector/src/main/java/pro/taskana/adapter/systemconnector/camunda/api/impl/CamundaTaskRetriever.java]: line 61 reads ... "exit from retrieveTerminatedCamundaTasks...."
Should read  "exit from retrieveFinishedCamundaTasks...."

File: taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/TaskanaConfigurationProperties.java (hunk 2 above)
Comment (gitgoodjhe) [reply taskana-adapter-camunda-outbox-rest/src/main/java/pro/taskana/adapter/camunda/TaskanaConfigurationProperties.java]: forgot auto-save

File: taskana-adapter-camunda-system-connector/src/main/java/pro/taskana/adapter/systemconnector/camunda/api/impl/CamundaTaskRetriever.java (hunk 4 above)
Comment (gitgoodjhe) [reply taskana-adapter-camunda-system-connector/src/main/java/pro/taskana/adapter/systemconnector/camunda/api/impl/CamundaTaskRetriever.java]: good catch

**Pull request merged**